spikeinterface/comparison/addtemplatesrecording.py

```diff
@@ -11,45 +11,57 @@ class AddTemplatesRecording(BaseRecording):
 
     Parameters
     ----------
-    TODO
+    sorting: BaseSorting
+        Sorting object containing all the units and their spike train.
+    templates: np.ndarray[n_units, time, n_channels]
+        Array containing the templates to inject for all the units.
+    nbefore: list[int] | None
```

Comment:
```suggestion
    nbefore: list[int] | int | None
```
In case of int, we can assume they are the same for all templates (e.g using `waveform_extractor.nbefore`)

Reply:
I'm adding it to the next commit! (There's a log of bugs in `get_traces` currently).